shapes/shapes.go

```diff
@@ -7,26 +9,36 @@ import (
 
 var pi = decimal.NewFromFloat(math.Pi)
 
+type Volumer interface {
+	CalculateVolume() decimal.Decimal
+}
+
+// Sphere is struct, which has only radius.
 type Sphere struct {
 	Radius decimal.Decimal
 }
 
+// Pyramid is struct, which has length, width, hieght
 type Pyramid struct {
 	Length, Width, Hieght decimal.Decimal
 }
 
+// Cylinder is struct, which has radius, hieght
 type Cylinder struct {
 	Radius, Hieght decimal.Decimal
 }
 
-func (Sphere *Sphere) Volume() decimal.Decimal {
+// CalculateVolume calculateVolume is realization calculateVolume() for Sphere
+func (Sphere *Sphere) CalculateVolume() decimal.Decimal {
 	return pi.Mul(decimal.NewFromFloat(4.0 / 3)).Mul(Sphere.Radius.Pow(decimal.NewFromFloat(3.0)))
 }
 
```

Comment:
```
func (s *Sphere) New(<params-to-create-sphere>) *Sphere  {
	return &Sphere{
               <init required fields>      
        }
}

```

 `New()` is a generic name for a function which building an objects(structs).  Need to and same functions for other shapes